ConfigVersionControl/git_version_control.py

```diff
@@ -223,4 +248,10 @@ def _should_ignore(self, file_path):
         # Ignore anything that starts with the system tests prefix
         # (unfortunately putting the system test prefix in the .gitignore doesn't work
         # because the git library always forces an add - it has a force flag, but it's not used)
-        return SYSTEM_TEST_PREFIX in file_path
\ No newline at end of file
+        return SYSTEM_TEST_PREFIX in file_path
+
+    def add_u(self):
```

Comment:
This makes sense from the description, but it's not self-evident from an interface point of view. `add_all_edited_files` would be better